packages/xo-server/src/api/proxy.js

```diff
@@ -106,6 +110,10 @@ deploy.params = {
     type: 'string',
   },
   network: {
+    type: 'string',
+    optional: true,
+  },
```

Comment:
This change is breaking the API.

Not a big issue but we should be aware of it and avoid it when possible.

Reply:
It's a recent change that's why i changed.